WordPress/Classes/ViewRelated/Post/WPStyleGuide+Posts.swift

```diff
@@ -0,0 +1,140 @@
+/// A WPStyleGuide extension with styles and methods specific to the Posts feature.
+///
+extension WPStyleGuide {
```

Comment:
I'm glad to see you converted these methods to Swift!

I've made some comments on places I think we could make it a bit more Swift-y without too much effort.